src/extension/wps/wps-core/src/test/java/org/geoserver/wps/other/NoArgWPSTest.java

```diff
@@ -0,0 +1,58 @@
+package org.geoserver.wps.other;
+
+import static org.junit.Assert.*;
+
+import java.io.InputStream;
+import java.io.StringWriter;
+import java.util.Map;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.transform.OutputKeys;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerConfigurationException;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+
+import org.apache.commons.io.IOUtils;
+import org.geoserver.data.test.SystemTestData;
+import org.geoserver.wps.WPSTestSupport;
+import org.junit.Test;
+import org.w3c.dom.Document;
+
+public class NoArgWPSTest extends WPSTestSupport  {
+
+	
+	
+	 @Override
+	    protected void registerNamespaces(Map<String, String> namespaces) {
+	        namespaces.put("feature", SystemTestData.BUILDINGS.getNamespaceURI());
+	    }
+	 
+	/**
+	 * This test runs a no-argument WPS Process and checks the result. 
+	 * @throws Exception
+	 */
+	 @Test 
+	public void NoArgumentProcessTest() throws Exception
+	{
```

Comment:
We use the other style of brace indents, check developers guide (amounts to java standard formatting with 100 col width)